banyan/resource_policy_tunnel_test.go

```diff
@@ -391,3 +395,119 @@ func testAccPolicy_tunnel_any_create_json(name string) string {
 }
 `, name)
 }
+
+func TestAccPolicy_tunnel_l4_optional_field(t *testing.T) {
+	var bnnPolicy policy.GetPolicy
+
+	rName := fmt.Sprintf("tf-acc-%s", acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum))
+
+	resource.Test(t, resource.TestCase{
+		Providers: testAccProviders,
+		//		CheckDestroy: testAccCheckPolicy_destroy(t, &bnnPolicy.ID),
```

Comment:
remove commented line?

Reply:
uncommented this line, it was supposed to be uncommented.